### ISSUE TITLE
Prediction assessment

### ISSUE THREAD

**Issue by aranas** [mlp2018/BagofWords#4]
Create split of training data (optimally in a cross-validation manner)
returns accuracy of prediction

**Comment by aranas** [comment]
splits should be random
option for replication through seeding.

**Comment by aranas** [comment]
- [x] implement cross-validator with random splitting

- [x] seeding

- [x] returns accuracy per fold

- [x] returns list of mis-classified exemplars

**Comment by aranas** [comment]
I used StratifiedKFold from the sklearn package to implement the cross-validation. It features shuffling before each split (shuffle=True) and also seeding (random_state=None). If we want achieve a split with 20% of the data left out for testing on each fold, we should do 5 splits.
The resulting array containing the predictions is sorted in the order of the shuffeled left-out data.

For now I then resort the results and the original labels according to the ids, but It hink it would actually make more sense to save them to use the test_index information when saving them to the results array in the first place. Because all the sorting and re-ordering is just confusing and creates room for error. 


- [ ] make cross-validation for-loop output predictions in the original ordering of the items

**Comment by aranas** [comment]
Have one initial split 90%-10% before cross-validation
